### PR TITLE
format error in CHANGELOG-1.14.md

### PR DESCRIPTION
**Pull request by qingsenLi**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
/kind cleanup

/sig release
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/lgtm
/ok-to-test
/priority important-soon

**Comment by neolit123** [comment]
@qingsenLi 
this is also a problem in the release-1.14 branch:
https://github.com/kubernetes/kubernetes/blob/release-1.14/CHANGELOG-1.14.md#apply-resources-from-a-directory-containing-kustomizationyaml-74140-liujingfang1

please send a cherry pick of this PR to that branch. here is info how:
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-release/cherry-picks.md#initiate-a-cherry-pick

thanks.

File: CHANGELOG-1.14.md [hunk 1]
```diff
@@ -272,7 +272,7 @@ filename | sha512 hash
...
 * Add a configuration field to shorten the timeout of validating/mutating admission webhook call. The timeout value must be between 1 and 30 seconds. Default to 30 seconds when unspecified.  ([#74562](https://github.com/kubernetes/kubernetes/pull/74562), [@roycaihw](https://github.com/roycaihw))
 * client-go: PortForwarder.GetPorts() now contain correct local port if no local port was initially specified when setting up the port forwarder ([#73676](https://github.com/kubernetes/kubernetes/pull/73676), [@martin-helmich](https://github.com/martin-helmich))
-* # Apply resources from a directory containing kustomization.yaml ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
+* Apply resources from a directory containing kustomization.yaml ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
     *         kubectl apply -k dir
     *         # Delete resources from a directory containing kustomization.yaml.
```
Comment (neolit123) [inline CHANGELOG-1.14.md]: /hold

the whole block needs to be formatted better:
```
        # Apply resources from a directory containing kustomization.yaml
        kubectl apply -k dir

        # Delete resources from a directory containing kustomization.yaml.
        kubectl delete -k dir

        # List resources from a directory containing kustomization.yaml
        kubectl get -k dir
```

i.e. not using bullets.

**Comment by qingsenLi** [comment]
/cc @neolit123 @tpepper 
I fix it,how about this:

File: CHANGELOG-1.14.md [hunk 2]
```diff
@@ -272,12 +272,10 @@ filename | sha512 hash
...
 * client-go: PortForwarder.GetPorts() now contain correct local port if no local port was initially specified when setting up the port forwarder ([#73676](https://github.com/kubernetes/kubernetes/pull/73676), [@martin-helmich](https://github.com/martin-helmich))
-* # Apply resources from a directory containing kustomization.yaml ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
-    *         kubectl apply -k dir
-    *         # Delete resources from a directory containing kustomization.yaml.
-    *         kubectl delete -k dir
-    *         # List resources from a directory containing kustomization.yaml
-    *         kubectl get -k dir
+* The examples in kubectl apply/get/delete are updated to support -k. ([#74140](https://github.com/kubernetes/kubernetes/pull/74140), [@Liujingfang1](https://github.com/Liujingfang1))
+    *   `kubectl apply -k dir` # Apply resources from a directory containing kustomization.yaml.
```
Comment (spiffxp) [inline CHANGELOG-1.14.md]: IMO the "title" / one-line sentence is the only that should show up in these change log entries.  I've edited the release notes block for #74140 accordingly

**Comment by spiffxp** [comment]
/cc @jeefy @dstrebel @onyiny-ang 
I'm not sure there is much point to updating this file right now, it's ultimately going to get replaced by the release notes currently being worked on by the v1.14 release notes team

**Comment by qingsenLi** [comment]
/cc @jeefy @dstrebel @onyiny-ang
I updated the PR，followed  #74140

**Pull request merged**